app/models/post.rb

```diff
@@ -3,18 +3,13 @@ class Post < ApplicationRecord
   validates_presence_of :body
   validates_presence_of :title
 
+  attr_accessor :my_sentiment
```

Comment:
Author: We let the author of the post guess what they think the sentiment is. But I see how that is a confusing name. What do you think about `author_sentiment`?